app/views/courses/subjects/_form_fields.html.erb

```diff
@@ -1 +1 @@
-<%= form.select :master_subject_id, course.selectable_master_subjects, {}, data: { qa: 'course__subjects' }, class: "govuk-select" %>
+<%= form.select :master_subject_id, options_for_select(course.selectable_master_subjects, @course.subjects.first&.id), {}, data: { qa: 'course__subjects' }, class: "govuk-select" %>
```

Comment:
The use of `first` is surprising to me in here. What's the reason for it?

Reply:
This might want to be split into another PR, it's setting the default for the dropdown to the first item in `@course.subjects`

Reply:
reverting this causes a test to fail

```
  1) Edit course subjects with a subject already set it should automatically select the current subject
     Failure/Error: expect(subjects_page).to have_select("course_master_subject_id", selected: english_subject.subject_name)
       expected to find visible select box "course_master_subject_id" that is not disabled with "English" selected but there were no matches. Also found "English Biology", which matched the 
selector but not all filters. Expected "English" to be selected found []
     # ./spec/features/courses/subjects/edit_spec.rb:78:in `block (3 levels) in <top (required)>'
                   
Finished in 29.6 seconds (files took 4.05 seconds to load)
439 examples, 1 failure, 1 pending                                   
                                                                                                
Failed examples:

rspec ./spec/features/courses/subjects/edit_spec.rb:76 # Edit course subjects with a subject already set it should automatically select the current subject
```